js/jest-tests/test/ScoreFrame.test.js

```diff
@@ -0,0 +1,24 @@
+// use named imports
+import { ScoreFrame } from '../src/ScoreFrame';
+
+describe('Frame compute', () => {
```

Comment:
Teño sentimentos cruzados neste test. O test é supercompleto, xa que fai unha matriz de tests en un só tests. Pero é un bicho considerable.  Estou máis inclinado a acustomarme a el, de todas formas.